packages/host/app/components/operator-mode/create-file-modal.gts

```diff
@@ -33,6 +59,33 @@ export default class CreateFileModal extends Component<Signature> {
             @onSelect={{this.onSelectRealm}}
           />
         </FieldContainer>
+        {{#if (eq @fileType.id 'card-instance')}}
+          <FieldContainer
+            @label='Inherits From'
+            class='field'
+            data-test-inherits-from-field
+          >
+            <div class='field-contents'>
+              {{#if this.selectedCatalogEntry}}
+                <Pill
+                  @inert={{true}}
+                  data-test-selected-type={{this.selectedCatalogEntry.ref.name}}
+                >
+                  {{this.selectedCatalogEntry.ref.name}}
```

Comment:
Is it correct to use `ref.name`? Not 100% sure but I think the module's name could also be `default` and then this display would say `default`.

Reply:
maybe `catalogEntry.title`?